### ISSUE TITLE
Per-target Tobit sigma — resolve CRPS/MCR tradeoff across targets

### ISSUE THREAD

**Issue by Polichinel** [views-platform/views-hydranet#44]
## Context

The sigma sensitivity sweep (2026-05-30, `pink_pirate_tobit_sigma_sweep_sweep`) established that no single Tobit sigma satisfies all three regression targets:

| sigma | lr_sb CRPS / MCR | lr_ns CRPS / MCR | lr_os CRPS / MCR |
|-------|------------------|------------------|------------------|
| 0.50 | 0.332 / **2.61** | 0.061 / **1.56** | 0.055 / **0.14** |
| 1.00 | **0.169** / 0.55 | **0.044** / 0.74 | 0.052 / 0.015 |

- **sigma=1.0** wins on CRPS (ranking quality) but leaves lr_os MCR at 0.015 (67× underprediction)
- **sigma=0.5** wins on MCR (magnitude calibration) — lr_os recovers to 0.14 (10× improvement) — but lr_sb CRPS doubles

The tradeoff maps to zero-inflation ratio: targets with more zeros (lr_os ~0.5% non-zero) need sharper sigma to learn from their rare uncensored cells, while targets with more signal (lr_sb ~5% non-zero) need softer sigma to avoid overfitting magnitudes.

## Proposed Solution

Make `loss_reg_sigma` accept a per-target dict, following the same pattern as `target_weights` (ADR-050):

```python
# Current (single sigma for all targets):
'loss_reg_sigma': 1.0

# Proposed (per-target sigma):
'loss_reg_sigma': {
    'lr_sb_best': 1.0,
    'lr_ns_best': 0.75,
    'lr_os_best': 0.5,
}
```

When `loss_reg_sigma` is a dict, each target gets its own `TobitLoss` instance with the corresponding sigma. When it's a float, all targets share one instance (backward compatible).

## Implementation

1. **Config:** `loss_reg_sigma: float | Dict[str, float]` — validator ensures all regression targets present when dict
2. **Loss creation:** `choose_loss()` returns either a single `TobitLoss` or a dict of per-target instances
3. **Training loop:** `_process_sequence()` selects the correct loss instance per target in the existing per-target loop
4. **Tests:** Per-target sigma produces different gradients per target; backward compat with float sigma

## Experiment Plan

Initial per-target sigma values derived from sweep results:

| Target | Zero-inflation | Optimal sigma | Rationale |
|--------|---------------|---------------|-----------|
| lr_sb_best | ~95% zeros | 1.0 | Best CRPS, MCR recovers with training |
| lr_ns_best | ~98% zeros | 0.75 | Balanced CRPS/MCR |
| lr_os_best | ~99.5% zeros | 0.5 | Only value producing meaningful MCR |

Single calibration run (80 lessons) to validate the hypothesis that per-target sigma combines the CRPS advantage of sigma=1.0 (for lr_sb) with the MCR advantage of sigma=0.5 (for lr_os).

## Acceptance Criteria

- [ ] `loss_reg_sigma` accepts both `float` and `dict[str, float]`
- [ ] Backward compatible — existing configs with `loss_reg_sigma: 1.0` work unchanged
- [ ] Per-target experiment validates: lr_sb CRPS ≤ 0.20, lr_os MCR ≥ 0.10
- [ ] All existing tests pass

## Depends On

- #36 Path A: Tobit loss (done)

## Blocks

- #37 Path E: Scheduled sampling (per-target sigma should be settled before adding teacher forcing complexity)

**Comment by Polichinel** [comment]
## Learnable Sigma Sweep Results (2026-06-01)

6-run sweep (2×3 grid): learnable [True/False] × 3 sigma initializations, 80 lessons each. Sigma values logged to wandb per lesson.

### Key finding: optimizer barely moves sigma in 80 lessons

| Run | Init (sb/ns/os) | Final σ | Max movement |
|-----|----------------|---------|-------------|
| Learnable | 1.0 / 0.75 / 0.5 | 1.02 / 0.77 / 0.53 | +5.6% (os) |
| Learnable | 1.0 / 1.0 / 1.0 | 1.02 / 1.02 / 0.96 | -3.7% (os) |
| Learnable | 1.5 / 0.75 / 0.25 | 1.51 / 0.77 / 0.26 | +5.9% (os) |

The optimizer consistently pushes os sigma up slightly (loosening) and sb sigma up slightly — but <6% movement total. Run-to-run variance in CRPS/MCR is larger than the learnable vs fixed effect.

From uniform init {1,1,1}, the optimizer moves os sigma toward 0.5 (correct direction) but would need ~1000 lessons at this rate.

### Recommendation

Use **fixed per-target sigma** `{sb: 1.0, ns: 0.75, os: 0.5}` for production. The `learnable_sigma` feature is validated and available for longer training runs where sigma may converge, but at 80 lessons the hand-picked values dominate.

### Status: issue #44 complete

- [x] Per-target sigma implemented and validated (3 sweeps)
- [x] Learnable sigma implemented and validated (1 sweep)
- [x] Wandb sigma logging working
- [x] Falsification audit passed (C-100 through C-103 resolved)
- [x] ADR-055 written, CIC updated
- [x] PR #47 merged
